packages/runtime/container-runtime/src/blobManager.ts

```diff
@@ -538,6 +572,12 @@ export class BlobManager {
             );
             const blobId = pathParts[2];
 
+            // GC tombstones these blobs
+            if (tombstone) {
+                this.tombstonedBlobs.add(blobId);
+                continue;
+            }
+
             // The unused blobId could be a localId. If so, remove it from the redirect table and continue. The
             // corresponding storageId may still be used either directly or via other localIds.
             if (this.redirectTable?.has(blobId)) {
```

Comment:
I thought we were going to do something with this logic, does this have to do with the redirect?

Reply:
Not sure what you mean. Could you provide some details on what something is?

Reply:
Oh yeah, I had forgotten to remove the if statement https://github.com/microsoft/FluidFramework/pull/12880#discussion_r1035287194

Reply:
oh yeah. Fixed it.